internal/controllers/aggregation/symphony.go

```diff
@@ -79,6 +78,27 @@ func (c *symphonyController) buildStatus(symph *apiv1.Symphony, comps *apiv1.Com
 		}
 	}
 
+	// Filter any values where one or more composition hasn't reached the corresponding state
```

Comment:
So we're still taking the max, but we're just not reflecting it on the status until all compositions actually reach said status, right?

Reply:
Yep exactly 👍